src/test/datascience/notebook/creation/notebookCreation.vscode.test.ts

```diff
@@ -59,86 +59,74 @@ suite('DataScience - VSCode Notebook - (Creation Integration)', function () {
+            traceInfo(`Quick Pick displayed to user`);
+            assert.isAtLeast(items.length, 2);
 
-                // If this is the first time this prompt was displayed, then select the second item (julia).
-                if (stub.callCount === 1) {
-                    return items[1];
-                }
+            // If this is the first time this prompt was displayed, then select the second item (javascript).
+            if (stub.callCount === 1) {
+                return items[1];
+            }
 
-                // Pick the first item, that will be us.
-                return items[0];
-            });
-            disposables.push({ dispose: () => stub.restore() });
+            // Pick the first item, that will be us.
+            return items[0];
+        });
+        disposables.push({ dispose: () => stub.restore() });
 
-            // Create a blank notebook & we should have a julia cell.
-            await createNotebookAndValidateLanguageOfFirstCell('julia');
-            assert.equal(stub.callCount, 1);
+        // Create a blank notebook & we should have a javascript cell.
+        await createNotebookAndValidateLanguageOfFirstCell('javascript');
+        assert.equal(stub.callCount, 1);
 
-            await closeActiveWindows();
+        await closeActiveWindows();
 
-            // Try again & this time select the first item from the list & we should end up with a python notebook.
-            await createNotebookAndValidateLanguageOfFirstCell(PYTHON_LANGUAGE.toLowerCase());
-            assert.equal(stub.callCount, 2);
-        });
-        test('Without 3rd party integration, do not display quick pick when selecting create blank notebook command', async function () {
-            assert.equal(creationOptions.registrations.length, 0);
-            assert.isUndefined(vscodeNotebook.activeNotebookEditor);
+        // Try again & this time select the first item from the list & we should end up with a python notebook.
+        await createNotebookAndValidateLanguageOfFirstCell(PYTHON_LANGUAGE.toLowerCase());
+        assert.equal(stub.callCount, 2);
+    });
+    test('Without 3rd party integration, do not display quick pick when selecting create blank notebook command', async function () {
+        assert.equal(creationOptions.registrations.length, 0);
+        assert.isUndefined(vscodeNotebook.activeNotebookEditor);
 
-            // Create a blank notebook & it should just work.
-            await createNotebookAndValidateLanguageOfFirstCell(PYTHON_LANGUAGE.toLowerCase());
-        });
-        test('Create Java & Julia Notebook using API', async function () {
-            await api.createBlankNotebook({ defaultCellLanguage: 'java' });
+        // Create a blank notebook & it should just work.
+        await createNotebookAndValidateLanguageOfFirstCell(PYTHON_LANGUAGE.toLowerCase());
+    });
+    test('Create javascript & powershell Notebook using API', async function () {
```

Comment:
these are the only two languages that seem to behave as expected, so it seems like this scenario is still mostly broken